_posts/2017/2017-12-31-avoid-double-negatives.md

```diff
@@ -48,13 +48,15 @@ end
 
 ## But why?
 
-Ruby’s conditional syntax is ‘truthy’. In Ruby’s case that means `nil` is equivalent to `false`. There is no need to do a negated `#nil?` check on an object as it is redundant as shown in the first two examples.
+Ruby’s conditional syntax is ‘truthy’, meaning `nil` is equivalent to `false`. Therefore a negated `#nil?` check on an object is redundant.
 
-The syntax of `!!` in the third example converts a value to boolean, but it is unnecessary and likely obscures your intention. If you _really_ want to do a `nil` check, you should use `#nil?` instead.
+The syntax of `!!` is two consecutive boolean ‘not’ operators, if takes any value and converts it into `true` or `false`. However, given Ruby's ‘truthy’ conditionals it is unnecessary and adds redundant code.
```

Comment:
I think there's a typo or two in this first sentence? Doesn't read properly.

Reply:
And when you say "given Ruby's `truthy` conditionals, what you mean is that: in Ruby, values can be 'truthy' or 'falsey' in and of themselves, so you don't need to convert objects into a boolean?

Reply:
I think you should just say that: `!!` conversts any value into a boolean of `true` or `false`. Don't mention the `not` operators thing. I think it confuses the sentence and takes away focus from your point/